lexicons/app/bsky/feed/detach.json

```diff
@@ -0,0 +1,27 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.detach",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "description": "Record defining post URIs detached from a root post. The record key (rkey) of the detach record must match the record key of the root post in question, and that record must be in the same repository.",
+      "record": {
+        "type": "object",
+        "required": ["targets", "updatedAt"],
+        "properties": {
+          "targets": {
+            "type": "array",
+            "maxLength": 1000,
```

Comment:
I'm inclined to start off with a lower maxLength. I think Paul threw out 50, but maybe 100 if we want some extra leeway?